### ISSUE TITLE
Jump between paired symbols

### ISSUE THREAD

**Issue by gaoqiangks** [James-Yu/LaTeX-Workshop#3645]
I switched from vim+vimtex to vscode+latexworkshop several weeks ago. I like your extension very much. There's a feature in vimtex that  "when the percentage symbol % is pressed in normal, you can jump to the symbol matching the one under the cursor". Certainly this is a feature built in vim. However, vim only allows you to jump between "(" and ")", or "{" and "}".  vimtex allows you to jump between "\begin{equation}" and "\end{equation}".   Can you please add this feature ?

**Comment by James-Yu** [comment]
I don’t understand what you mean. Please fill in all contents in the feature request template.

**Comment by jlelong** [comment]
I think the command `latex-workshop.navigate-envpair` does what you want. See https://github.com/James-Yu/LaTeX-Workshop/wiki/Environments#navigating-and-selecting

You can even define a shortcut for this command. See at the bottom of page https://code.visualstudio.com/docs/getstarted/keybindings

**Comment by gaoqiangks** [comment]
> I think the command `latex-workshop.navigate-envpair` does what you want. See https://github.com/James-Yu/LaTeX-Workshop/wiki/Environments#navigating-and-selecting
> 
> You can even define a shortcut for this command. See at the bottom of page https://code.visualstudio.com/docs/getstarted/keybindings

Thank you very much!

**Comment by TryerGit** [comment]
> I think the command `latex-workshop.navigate-envpair` does what you want

This does work for some environment pairs. In particular this does not work for the following structures/packages:

```
\usepackage{ifthen} 
\newif\ifanswer 
```

The typical usage then within the `.tex` is:

```tex
\begin{document}
\usepackage{ifthen} 
\newif\ifanswer 
\answertrue % this produces the answer key, set this to \answerfalse for the qQuestion paper only with no answers

Question : How many hours are in a day?
\ifanswer
    24
\else
\bigskip
\fi
\end{document}
```

From cursor on `\ifanswer`, it would be great if it were possible to navigate to `\else` and then subsequently to `\fi` and then cycle back.

It may not be easy to implement -- I do not know. A Vim extension manages to accomplish this for LaTeX documents. Please see [here](https://github.com/lervag/vimtex/issues/2609#event-8116014295)

**Comment by jlelong** [comment]
For the time being, the command `latex-workshop.navigate-envpair` only works for environments, ie `\begin{...} ... \end{...}` constructions.

**Comment by jlelong** [comment]
The feature request described in the above post https://github.com/James-Yu/LaTeX-Workshop/issues/3645#issuecomment-1383074523 has been implemented and is available in release 9.8.0.

You can use `latex-workshop.navigate-envpair` to cycle between `\if*`, `\else`, `\fi`.
